modules/op-configuring-basic-authentication-for-git.adoc

```diff
@@ -0,0 +1,84 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-configuring-basic-authentication-for-git_{context}"]
+= Configuring basic authentication for Git
+
+For basic authentication, {pipelines-shortname} creates a `~/.gitconfig` file containing the credentials specified in the secret, before executing any steps in a task run or a pipeline run. When the steps execute, {pipelines-shortname} uses those credentials to retrieve the pipeline resources specified in a run.
+
+[discrete]
+.Procedure
+
+To configure a Git secret of the `basic-auth` type:
+
+. In the `secret.yaml` file, specify the username and password required to access the target Git repository.
++
+[NOTE]
+====
+For GitHub, authentication using plain password is deprecated. Instead, use a link:https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token[personal access token].
+====
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: basic-user-pass
+  annotations:
+    tekton.dev/git-0: https://github.com
+type: kubernetes.io/basic-auth
+stringData:
+  username: <cleartext username>
+  password: <cleartext password or personal access token>
+----
++
+. In the `serviceaccount.yaml` file, associate the secret with the appropriate service account.
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: build-bot
+secrets:
+  - name: basic-user-pass
+----
++
+. In the `run.yaml` file, associate the service account with a task run or a pipeline run.
++
+* To associate the service account with a task run:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: build-push-task-run-2
+spec:
+  serviceAccountName: build-bot
+  taskRef:
+    name: build-push
+----
++
+* To associate the service account with a pipeline run:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: demo-pipeline
+  namespace: default
+spec:
+  serviceAccountName: build-bot
+  pipelineRef:
+    name: demo-pipeline
+----
++
+. Execute the run.
++
+[source,terminal]
+----
+$ kubectl apply --filename secret.yaml,serviceaccount.yaml,run.yaml
+----
```

Comment:
Please apply suggestions from this file to similar instances in the remaining files. (I won't be repeating those suggestions.)